triagebot.toml

```diff
@@ -4,9 +4,23 @@ allow-unauthenticated = [
     "good-first-issue"
 ]
 
-[assign]
-
 # Allows shortcuts like `@rustbot ready`
 #
 # See https://github.com/rust-lang/triagebot/wiki/Shortcuts
 [shortcut]
+
+[assign]
+contributing_url = "https://github.com/rust-lang/rust-clippy/blob/master/CONTRIBUTING.md"
```

Comment:
What does this field do? Like where is it mentioned? :)

Reply:
In the welcome message to new users.  It looks like [this](https://github.com/rust-lang/rust-clippy/pull/9962#issuecomment-1327991187). There is documentation for the assignment options at https://github.com/rust-lang/triagebot/wiki/Assignment#additional-new-pr-trigger-options.